docs/governance/decred-constitution.md

```diff
@@ -1,48 +1,41 @@
-
-* *Universal Fungibility* - Universal fungibility is fundamental to *Decred* being a store of value and attacks against it shall be actively monitored and countermeasures pursued as necessary.
+* *Fixed Finite Supply* - Issuance is finite and the total maximum number of coins in Decred shall not change. The total maximum supply for Decred is 20,999,999.99800912 coins, with a per-block subsidy that adjusts every 6,144 blocks (approximately 21.33 days) by reducing by a factor of 100/101. The genesis block subsidy starts at 31.19582664 coins.
+* *Universal Fungibility* - Universal fungibility is fundamental to Decred being a store of value and attacks against it shall be actively monitored and countermeasures pursued as necessary.
 
 ------
 
 ## Blockchain Governance
 
-- Governance of the network occurs directly through the blockchain via hybridization of a block's proof-of-work ("PoW") with its proof-of-stake ("PoS"). PoS voters, also known as stakeholders, can effectively override PoW contributors, known as miners, if 60% or more of the stakeholders vote against a particular block created by a miner.
-- A lottery system is used to determine which stakeholders vote on each block and collect a subsidy.
+- [Governance](https://docs.decred.org/governance/introduction-to-decred-governance/) of the network occurs directly through the blockchain via hybridization of a block's proof-of-work ("PoW") with its proof-of-stake ("PoS"). PoS voters, also known as stakeholders, can effectively override PoW contributors, known as miners, if 50% or more of the stakeholders vote against a particular block created by a miner.
 - To be a stakeholder, one must purchase one or more tickets, which entails locking a specific amount of coins. The amount of DCR that must be locked, also known as the ticket price, fluctuates as the system targets 40,960 tickets in the live pool.
-- After waiting for the ticket to mature, the ticket is entered into a lottery that runs once per block where five winning tickets gain the ability to vote on the previous block.
+- In each block, 5 tickets are called to vote, determined by a lottery [system](https://docs.decred.org/proof-of-stake/proof-of-stake/). When a ticket is called, a nominated wallet must actively respond with a vote. For a block to be accepted by the network it must include votes from at least 3 of the 5 tickets called. The blockchain cannot be extended without the active participation of stakeholders.
 - Stakeholders must wait an average of 28 days (8,192 blocks) to vote their tickets, and during this time the coins used to purchase the ticket remain locked. The wait may be much longer or shorter than the average of 28 days because the ticket selection process is pseudorandom. Tickets expire after approximately 142 days (40,960 blocks).
```

Comment:
Should include a "~" before 40,960 blocks.

Reply:
40,960 is not an approximation, that is exactly the number of blocks it takes for tickets to expire. 142 days is an approximation.

Reply:
I would switch the placement then, since approximations should come after exact amounts. 

Revised: "Tickets expire after 40,960 blocks (~142 days).

Reply:
Done.